test/skimNtuple_HHbtag.cpp

```diff
@@ -170,6 +170,69 @@ void setScaleFactor(std::map<std::string, std::vector<float>>& result_, const st
 }
 
 
+// Legacy triggers take precedence over Single-Tau and MET triggers
+std::unordered_map<std::string,bool> assignTriggerRegions(int ptype, std::string period,
+														  float pt1, float pt2, float eta1, float eta2,
+														  float eleEtaMax, float muEtaMax, float tau_thresh)
+{
+  bool Legacy_region    = false;
+  bool SingleTau_region = false;
+  bool MET_region       = false;
+  
+  if (ptype == 0) { //mutau
+	bool SingleLepton_validity = false;
+	bool CrossLepton_validity  = false;
+	bool SingleTau_validity    = false;
+	
+	if(period=="2018") {
+	  SingleLepton_validity = pt1 >= 33. and eta1 < muEtaMax;
+	  CrossLepton_validity	= pt1 >= 25. and eta1 < 2.1 and pt2 >= 35. and eta2 < 2.1;
+	  SingleTau_validity	= pt2 >= tau_thresh and eta2 < 2.1;
+	}
+	else if(period=="2017") {
+	  SingleLepton_validity = pt1 >= 28. and eta1 < muEtaMax;
+	  CrossLepton_validity	= pt1 >= 21. and eta1 < 2.1 and pt2 >= 32. and eta2 < 2.1;
+	  SingleTau_validity	= pt2 >= tau_thresh and eta2 < 2.1;
+	}
+	else if (period=="2016preVFP" or period=="2016postVFP") {
+	  SingleLepton_validity = pt1 >= 25. and eta1 < muEtaMax;
+	  CrossLepton_validity	= pt1 >= 20. and eta1 < 2.1 and pt2 >= 25. and eta2 < 2.1;
+	  SingleTau_validity	= pt2 >= tau_thresh and eta2 < 2.1;
+	}
+
+	Legacy_region = SingleLepton_validity or CrossLepton_validity;
+	SingleTau_region = SingleTau_validity and !Legacy_region;
+  }
+
+  else if (ptype == 1) { //etau
+	if(period=="2018" or period=="2017") {
+	  bool SingleLepton_validity = pt1 >= 33. and eta1 < eleEtaMax;
+	  bool CrossLepton_validity  = pt1 >= 25. and eta1 < 2.1 and pt2 >= 35. and eta2 < 2.1;
+	  bool SingleTau_validity    = pt2 >= tau_thresh and eta2 < 2.1;
+
+	  Legacy_region = SingleLepton_validity or CrossLepton_validity;
+	  SingleTau_region = SingleTau_validity and !Legacy_region;
+	}
+	else if (period=="2016preVFP" or period=="2016postVFP") {
+	  Legacy_region    = pt1 >= 26. and eta1 < 2.1;
+	  SingleTau_region = pt2 >= tau_thresh and !Legacy_region;
+	}
+  }
+
+  else if (ptype == 2) { //tautau
+	Legacy_region = pt1 >= 25. and pt2 >= 21. and eta1 < 2.1 and eta2 < 2.1;
```

Comment:
HLT_DoubleMediumCombinedIsoPFTau35_Trk1_eta2p1_Reg_v --> pt1 and pt2 threshold should be 40